codecov.yaml

```diff
@@ -1,13 +1,15 @@
 coverage:
+  precision: 2
+  round: up
+  range: "45...70"
   status:
     project:
       default:
         target: auto
-        # allow coverage to drop by this amount and still post success
-        threshold: 0.5%
-        base: auto
-        branches:
-          - master
-        if_not_found: success
-        if_ci_failed: error
-        only_pulls: false
+        threshold: 5%
+        informational: true
+    patch:
+      default:
+        target: auto
+        threshold: 5%
+        informational: true
```

Comment:
Same here, I would propose to comment `informational`.

Notice that even if CI fails, this does not block maintainers from merging the change because Codecov testing is not configured as mandatory for the github repo.

Reply:
See previous comment.  I'll remove this.  I'm ok with seeing a red x for a 5% drop.